Scripts/SL-SupportHelpers.lua

```diff
@@ -85,17 +85,12 @@ end
 -- use StepManiaVersionIsSupported() to check if Simply Love supports the version of SM5 in use
 
 StepManiaVersionIsSupported = function()
-	-- SM5.0.12 is supported (latest stable release)
-	-- SM5.1.x is supported
-	-- SM5.2 is not supported because it saw significant
-	--       backwards-incompatible API changes and is now abandoned
+	-- SM5.0.12, SM5.1.x and OutFox are no longer supported
 	if IsStepMania() then
-		return IsProductVersion(5, 0, 12) or IsProductVersion(5, 1)
+		return false
```

Comment:
I think you can just kill the IsStepMania and IsOutFox conditions altogether since the else clause of IsItgmania will already cover it.

Reply:
oh yeah it does. i'll remove those then